webapp/i18n/en.json

```diff
@@ -6,8 +6,10 @@
   "BoardComponent.new": "+ New",
   "BoardComponent.no-property": "No {property}",
   "BoardComponent.no-property-title": "Items with an empty {property} property will go here. This column cannot be removed.",
+  "BoardComponent.openMenu": "Open Menu",
+  "BoardComponent.selectColor": "Select {color} Color",
   "BoardComponent.show": "Show",
-  "BoardPage.syncFailed": "Board may be deleted or access revoked.",
```

Comment:
This property is removed automatically by running the following command. `npm run i18n-extract`

Reply:
Thats ok :)